apps/meeting/src/providers/AppStateProvider.tsx

```diff
@@ -59,11 +61,36 @@ export function AppStateProvider({ children }: Props) {
   const [isWebAudioEnabled, setIsWebAudioEnabled] = useState(true);
   const [priorityBasedPolicy, setPriorityBasedPolicy] = useState<VideoPriorityBasedPolicy| undefined>(undefined);
   const [enableSimulcast, setEnableSimulcast] = useState(false);
-  const [blurOption, setBlur] = useState(BlurValues.blurDisabled);
   const [theme, setTheme] = useState(() => {
     const storedTheme = localStorage.getItem('theme');
     return storedTheme || 'light';
   });
+  const [videoTransformCpuUtilization, setCpuPercentage] = useState(VideoFiltersCpuUtilization.CPU40Percent);
+  const [imageBlob, setImageBlob] = useState<Blob | undefined>(undefined);
+  
+  useEffect(() => {
+    /* Load a canvas that will be used as the replacement image for Background Replacement */
+    async function loadImage(){
+      const canvas = document.createElement('canvas');
+      canvas.width = 500; 
+      canvas.height = 500;
+      const ctx = canvas.getContext('2d');
+      if(ctx != null){
```

Comment:
This PR still uses a non-strict comparison like `!=` and `==` (e.g., in `apps/meeting/src/components/DeviceSelection/CameraDevices/VideoTransformDropdown.tsx`). Please convert them to `!==` and `===`.

Reply:
sorry, completely missed those. should be fixed now in VideoInputTransformControl as well.